File: JPA/ex1-hello-jpa/src/main/java/jpa/exam/Member.java
package jpa.exam;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
public class Member {
    @Id
    private Long id;
    private String name;

    public Long getId() {
        return id;
    }

    public void setId(final Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(final String name) {
        this.name = name;
    }
}

File: Kotlin/spring-project/src/main/kotlin/com/example/demo/domain/Member.kt
package com.example.demo.domain

import javax.persistence.*

@Entity
class Member(
        @Column(unique = true)
        var name: String,

        @Column
        var age: Int,

        @ManyToMany
        @JoinTable(
                name = "asd",
                inverseJoinColumns = [JoinColumn(name = "aaa_id")]
        )
        var a: List<Team> = arrayListOf()
) : AbstractJpaPersistable()